nautobot/docs/release-notes/version-1.4.md

```diff
@@ -161,6 +161,7 @@ A new configuration setting, [`STRICT_FILTERING`](../configuration/optional-sett
 
 The `settings_and_registry` default context processor was changed to purely `settings` - the (large) Nautobot application registry dictionary is no longer provided as part of the render context for all templates by default. Added a new `registry` template tag that can be invoked by specific templates to provide this variable where needed.
 
+<!-- towncrier release notes start -->
```

Comment:
TODO: When before we merge this we should back out the changelog elements already in v1.4.3 to be fragments in the changelog directory. Otherwise someone is going to have to merge them after running towncrier on release.